recipe/meta.yaml

```diff
@@ -7,18 +7,10 @@ package:
 source:
   fn: iris-v{{ version }}.tar.gz
   url: https://github.com/SciTools/iris/archive/v{{ version }}.tar.gz
-  sha256: 62e4cb152366e21aa845f1fbf0520abda39ccc7c5952e6a434eed5cd9f2cc140
-  patches:
-    # Issue reported in https://github.com/SciTools/iris/issues/2004.
-    # We work around this in build.sh.
-    - no_compile_setup.patch  # [osx]
```

Comment:
Sorry @lbdreyer I was not specific on what patches you should drop. This one must stay. I meant only the other two below because they are upstream.

Maybe we should upstream this on too. Not sure how though b/c it is coded as a workaround at the moment. Maybe the `no_compile` could be a `setup.py` option?